src/layout/Footer.tsx

```diff
@@ -11,6 +11,12 @@ import { LanguageSelector } from './header/LanguageSelector';
 export const Footer = () => {
   const { t } = useTranslation();
 
+  const StyledArrowLinkContainer = styled(Box)({
+    display: 'flex',
+    gap: '0.5rem',
+    marginBottom: '0.3rem',
+  });
```

Comment:
Når denne ligger her vil den genereres på nytt for hver render. Derfor ønsker vi å ha den over `export const Footer`, slik at den initialiseres bare en gang, og dermed gjenbrukes flere ganger